spec/indexers/work_indexer_spec.rb

```diff
@@ -290,6 +290,77 @@
       end
     end
 
+    describe "with OCR" do
+      let(:assets) do
+        [3,2,1].map { |page| create(
+          :asset,
+          position: page,
+          published: true,
+          hocr: hocr
+          ) }
+      end
+
+      let (:hocr) { File.read('spec/test_support/hocr_xml/hocr.xml')}
+      let(:expected_hocr) {
+        ["CAUTION All units must be connected as above before the Power Supply is connected."] * 3
+      }
+
+      # transcription : en
+      # translation   : en
+      let(:english_only_work) do
+        create(:public_work, language: ['English'], members: assets )
+      end
+
+      # transcription : agnostic
+      # translation   : en
+      let(:bilingual_work) do
+        create(:public_work, language: ['English', 'German'], members: assets)
+      end
+
+      # transcription : de
+      # translation   : en
+      let(:german_only_work) do
+        create(:public_work, language: ['German'], members: assets )
+      end
+
+      let(:nil_members_work) do
+        create(:public_work, members: [])
+      end
+
+      before do
+        allow(nil_members_work).to receive(:members).and_return(nil)
+      end
+
+      let(:output_hash) { WorkIndexer.new.map_record(language_test_work) }
+      let(:english) { output_hash["searchable_fulltext_en"] }
+      let(:german)  { output_hash["searchable_fulltext_de"] }
+      let(:unsure)  { output_hash["searchable_fulltext_language_agnostic"] }
+      describe "text known to be in English" do
+        let(:language_test_work) { english_only_work }
```

Comment:
What is the point of definining a `let (:english_only_work)` above, only to use it once here?  Just define it inline here, less indirection, makes the tests easier to read? 

    let(:language_test_work) { create(:public_work, language: ['English'], members: assets )  }

Same for the others?

Reply:
Done!